examples/user-notification.amp.html

```diff
@@ -229,6 +229,37 @@
      <button on="tap:amp-user-notification3.dismiss">I accept</button>
   </amp-user-notification>
 
+  <amp-user-notification
+          layout=nodisplay
+          id="amp-user-notification4"
+          always-show
+          data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
since the default is to "store" the info, is there any gain for an attribute like `persistent-dismissal` having a true/false state (true by default) and not just use a an attribute like `bypass-cache` (standalone, no value)